tests/parm/diag_table_bmark_v16_merra2_template

```diff
@@ -1,5 +1,5 @@
-YMD.00Z.ATMRES.64bit.non-mono
```

Comment:
Come to think of this, I think this will fail.
`@[YMD]` is not defined, it is constructed from `SYEAR`, `SMONTH`, and `SDAY`.
The `sed` function was building it.
I'll push a commit that fixes this.

Reply:
13a282b will fix this.